modules/rosa-nodes-machine-pools-local-zones.adoc

```diff
@@ -0,0 +1,76 @@
+
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-nodes-machinepools-configuring.adoc
+:_content-type: CONCEPT
+[id="rosa-nodes-machine-pools-local-zones_{context}"]
+= Local Zones
+
+Use the following steps to configure Local Zones for machine pools.
+
+[IMPORTANT]
+====
+AWS Local Zones are supported on Red Hat OpenShift Service on AWS 4.12, and must be enabled by opening a support case with xref:rosa_architecture/rosa-getting-support.html#rosa-getting-support_rosa-getting-support [Red Hat Support].
```

Comment:
The link for 'Red Hat Support' can't reach

Reply:
Because this is a preview, the links won't work as intended. The link will work once it's live.

Reply:
I think it's the the missing ../ and space between the xref and the title that's causing the link to fail. Give this a try:

xref:../rosa_architecture/rosa-getting-support.html#rosa-getting-support_rosa-getting-support[Red Hat Support].